executor/infoschema_reader.go

```diff
@@ -153,3 +157,104 @@ func dataForViews(ctx sessionctx.Context, schemas []*model.DBInfo) ([][]types.Da
 	}
 	return rows, nil
 }
+
+// DDLJobsReaderExec executes DDLJobs information retrieving
+type DDLJobsReaderExec struct {
```

Comment:
Too much code is duplicate with `ShowDDLJobQueriesExec`, I suggest refactoring it.

Reply:
Add a struct `DDLJobExecInitializer` to reuse the DDLJobs initial logic in `Open()`, and other operation has some different code should bound to their executor which would not be refactord.